audbackend/core/backend/base.py

```diff
@@ -419,17 +416,19 @@ def ls(
             if self.exists(path):
                 paths = [path]
             else:
-                if not suppress_backend_errors:
-                    # since the backend does no longer raise an error
-                    # if the path does not exist
-                    # we have to do it
-                    try:
-                        raise utils.raise_file_not_found_error(path)
-                    except FileNotFoundError as ex:
-                        raise BackendError(ex)
                 paths = []
 
         if not paths:
+
+            if path != '/' and not suppress_backend_errors:
+                # since the backend does no longer raise an error
+                # if the path does not exist
+                # we have to do it
```

Comment:
The first part of the comment will not make much sense in the future, so maybe change to:

```suggestion
                # if the path does not exist
                # we raise an error
```

If I remember correctly, this behavior makes sense as we cannot have empty sub-paths on the backend, correct?